### ISSUE TITLE
[igNotifier] incorrect return type of `isVisible` method

### ISSUE THREAD

**Issue by tkiryu** [IgniteUI/ignite-ui#1904]
### Description  
I ran into a TypeScript check error when I implement like below.
```
    const $notifier = $('#notifier');

    if ($notifier.igNotifier('isVisible')) { // <- error
        // do something
    }
```

I found TypeScript definition file defines that `isVisible` method returns **void**.
```
// @types/ignite-ui@18.2.0 index.d.ts 
interface JQuery {
	igNotifier(methodName: "changeLocale"): void;
	igNotifier(methodName: "notify", state: Object, message?: string): void;
	igNotifier(methodName: "isVisible"): void;
        ...
```

### Steps to reproduce  
Implement like below.
```
    const $notifier = $('#notifier');

    if ($notifier.igNotifier('isVisible')) {

    }
```
### Result  
An error occur.

### Expected result  
An error should not occur.

### Attachments  
None

**Comment by ymita** [comment]
@SlavUI 
Do you know when we will update the npm package that includes this fix?
CC: @tkiryu

**Comment by SlavUI** [comment]
@bazal4o do you have idea when will this be available ?

**Comment by kdinev** [comment]
@SlavUI @ymita The fix is already available in the SRs. It will be available in the 19.1 RTM as well.
